### PR TITLE
Leftovers from pr #13339

### PR DESCRIPTION
**Pull request by Raroun**
#13339 has left some residue, which I would like to clean up and clear up any misunderstandings between "first and last name" and display name.
I am aware that we do not usually change source strings because the translators at transifex have to start over, but I have not found any other way.

### CODE REVIEW

**Comment by Raroun** [comment]
i don´t know why messages.po is failing -.-

File: src/Module/Admin/Site.php [hunk 1]
```diff
@@ -462,7 +462,7 @@ protected function content(array $request = []): string
...
 			'$enable_multi_reg'       => ['enable_multi_reg', DI::l10n()->t('Enable multiple registrations'), !DI::config()->get('system', 'block_extended_register'), DI::l10n()->t('Enable users to register additional accounts for use as pages.')],
 			'$enable_openid'          => ['enable_openid', DI::l10n()->t('Enable OpenID'), !DI::config()->get('system', 'no_openid'), DI::l10n()->t('Enable OpenID support for registration and logins.')],
-			'$enable_regfullname'     => ['enable_regfullname', DI::l10n()->t('Enable Fullname check'), !DI::config()->get('system', 'no_regfullname'), DI::l10n()->t('Enable check to only allow users to register with a space between the first name and the last name in their full name.')],
+			'$enable_regfullname'     => ['enable_regfullname', DI::l10n()->t('Enable Fullname check'), !DI::config()->get('system', 'no_regfullname'), DI::l10n()->t('Check the box to only allow users to register which display name consists of at least two strings.')],
```
Comment (MrPetovan) [inline src/Module/Admin/Site.php]:
```suggestion
			'$enable_regfullname'     => ['enable_regfullname', DI::l10n()->t('Enable full name check'), !DI::config()->get('system', 'no_regfullname'), DI::l10n()->t('Prevents users from registering with a display name with fewer than two parts separated by spaces.')],
```

File: static/settings.config.php [hunk 2]
```diff
@@ -161,7 +161,7 @@
 		'min_memory' => 0,
 
 		// no_regfullname (Boolean)
-		// Allow pseudonyms (true) or enforce a space between first name and last name in Full name, as an anti spam measure (false).
+		// Allow pseudonyms (true) or allow users to register which display name consists of at least two strings, as an anti spam measure (false).
```
Comment (MrPetovan) [inline static/settings.config.php]:
```suggestion
		// Allow users to register with a display name comporting fewer than two parts separated by spaces.
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Changing source strings is fine if the meaning changes like in this case. On the other hand typos shouldn't be fixed in source strings.

**Comment by MrPetovan** [comment]
Now that you've included my suggested changes, you should re-generate the `messages.po` file using the `bin/run_xgettext.sh` script.

**Comment by Raroun** [comment]
thanks, i was unsure because messages.po allways fails in the code checks.

**Comment by MrPetovan** [comment]
It comes from a discrepancy between the gettext version that's used in the Continuous Integration pipeline. 🤷‍♂️

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**